iina/HistoryWindowController.swift

```diff
@@ -95,22 +95,32 @@ class HistoryWindowController: NSWindowController, NSOutlineViewDelegate, NSOutl
     historyData[key]!.append(entry)
   }
 
+  private func removeAfterConfirmation(_ entries: [PlaybackHistory]) {
+    Utility.quickAskPanel("delete_history", sheetWindow: window) { respond in
+      guard respond == .alertFirstButtonReturn else { return }
+      HistoryController.shared.remove(entries)
+    }
+  }
+
   // MARK: Key event
 
   override func keyDown(with event: NSEvent) {
-    let commandKey = NSEvent.ModifierFlags.command
-    if event.modifierFlags.intersection(.deviceIndependentFlagsMask) == commandKey  {
+    let flags = event.modifierFlags.intersection(.deviceIndependentFlagsMask)
+    if flags == .command  {
       switch event.charactersIgnoringModifiers! {
-      case "f":
-        window!.makeFirstResponder(historySearchField)
-      case "a":
-        outlineView.selectAll(nil)
-      default:
-        break
+        case "f":
```

Comment:
Let's try to keep the code consistency -- align `case` with `switch`

Reply:
Oh I didn't even notice this! Thanks.